pkg/daemon/multus/resources.go

```diff
@@ -0,0 +1,290 @@
+/*
```

Comment:
What about naming this file validationtest.go?

Reply:
a file called `validation.go` already exists. This file handles the resources that are started by the test, similar to the `spec.go` files for the operator.